manila/share/drivers/netapp/dataontap/client/client_cmode.py

```diff
@@ -2179,7 +2202,7 @@ def configure_cifs_options(self, security_service):
         elif security_service.get('defaultadsite', None):
             api_args = {'mode': 'site'}
         else:
-            return
+            api_args = {'mode': 'all'}
```

Comment:
although it's not a change from this PR, but I notice that the errors are not thrown but only logged
is it always ok to do so, when this `configure_cifs_options` function called when updating security service's server and deafult-ad-site? If the discovery mode failed to be modified, would the changes in security service take effect? I would say, the client should know whether the intended changes are really applied correctly